src/mono/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -410,6 +410,9 @@ public override Assembly GetSatelliteAssembly(CultureInfo culture, Version? vers
             return res;
         }
 
+        internal const string UnknownStringMessageInRAF = "Returns <Unknown> for modules with no file path";
+
+        [RequiresAssemblyFiles(UnknownStringMessageInRAF)]
```

Comment:
The message is different than ref `"This member throws an exception for assemblies embedded in a single-file app"`

Reply:
You're right, these two have the wrong message.